service_person_stamdata_udvidet/udvidet_person_stam_data_lokal.py

```diff
@@ -17,13 +17,10 @@
 
 
 # Service
-service_url = (
-        "https://prod.serviceplatformen.dk/service/CPRInformation/"
-        "CPRInformation/1"
-    )
+service_url = "https://prod.serviceplatformen.dk/service/CPR/PersonBaseDataExtended/4"
 
 
-def get_citizen(service_uuids, certificate, cprnr):
+def get_citizen(service_uuids, certificate, cprnr, service_url=service_url):
     """The function returnes a citizen dict from the
```

Comment:
We may want to add both the URI for production and test to the codebase and change the parameter to a toggle between them, e.g. 

`get_citizen(service_uuids, certificate, cprnr, production=False):`

Reply:
:+1: